lang/lualanes/Makefile

```diff
@@ -1,5 +1,5 @@
 #
-# Copyright (C) 2007-2014 OpenWrt.org
+# Copyright (C) 2007-2024 OpenWrt.org
```

Comment:
Why?

Reply:
Is it incorrect to update the copyright date to reflect the latest year of changes? I'll happily set it back to 2014 if that is the convention.

Reply:
Shouldnt you do that only if you are affiliated to OpenWrt project? :-)

Reply:
I'm pretty new to this, but still trying to help contribute for the community. I'll resubmit with having changed back to '2014'.